libnymea-zigbee/zigbeenetworkdatabase.cpp

```diff
@@ -287,6 +304,18 @@ bool ZigbeeNetworkDatabase::initDatabase()
         createIndices("attributesIndex", "attributes", "clusterId, attributeId");
     }
 
+    if (!m_db.tables().contains("bindings")) {
+        createTable("bindings", "(sourceAddress TEXT NOT NULL, "
```

Comment:
I am not arguing against this. Question is if it's worth to break experimental setups by changing this, although to be fair it shouldn't be many

Reply:
the reason why I named it sourceAddress is because that's the wording in the ZigBee spec... In hindsight sourceIeeeAddress would probably have been a little better, I guess, but I wouldn't break every experimental setup for this as it's not *bad* either as it is.